tasks/sync_repo_languages.py

```diff
@@ -46,45 +45,50 @@ def run_impl(
             and len(desired_languages_intersection) == 0
         ) or manual_trigger
 
+        if not should_sync_languages:
+            return {"successful": True, "synced": False}
+
+        log_extra = dict(
+            owner_id=repository.ownerid or "",
+            repository_id=repository.repoid,
```

Comment:
both of these are now covered by the `LogContext`, so I would not duplicate them here.